h/templates/deform/mapping_item.jinja2

```diff
@@ -1,36 +1,54 @@
-{%- if not field.widget.hidden -%}
-<div class="form-group form-field{% if field.error %} form-field-error{% endif %}"
-     {%- if field.description -%}
-     title="{{ _(field.description) }}"
-     {% endif %}
-     id="item-{{ field.oid }}">
-{% endif -%}
+{% if feature('activity_pages') and not field.widget.hidden %}
```

Comment:
In my initial implementation the structure of the HTML was different with and without the feature flag. That's no longer really the case, so I might be able to simplify this to just changing a few CSS classes depending on whether the feature flag is set.